src/web-auth/hosted-pages.js

```diff
@@ -95,4 +101,33 @@ HostedPages.prototype.signupAndLogin = function(options, cb) {
   });
 };
 
+HostedPages.prototype.getSSOData = function(withActiveDirectories, cb) {
+  var url;
+  var params = '';
+
+  if (typeof withActiveDirectories === 'function') {
+    cb = withActiveDirectories;
+    withActiveDirectories = false;
+  }
+
+  assert.check(withActiveDirectories, {
+    type: 'boolean',
+    message: 'withActiveDirectories parameter is not valid'
+  });
+  assert.check(cb, { type: 'function', message: 'cb parameter is not valid' });
+
+  if (withActiveDirectories) {
+    params =
+      '?' +
+      qs.stringify({
+        ldaps: 1,
+        client_id: this.baseOptions.clientID
+      });
+  }
+
+  url = urljoin(this.baseOptions.rootUrl, 'user', 'ssodata', params);
+
+  return this.request.get(url, { noHeaders: true }).withCredentials().end(responseHandler(cb));
```

Comment:
why `noHeaders=true`?

Reply:
just copied from v8: https://github.com/auth0/auth0.js/blob/v8/src/authentication/index.js#L382

Reply:
but what does passing this parameter do?

Reply:
doesn't send the headers https://github.com/auth0/auth0.js/blob/master/src/helper/request-builder.js#L79

Reply:
This is also removing the telemetry header.. What's the reason for not sending any headers now? I know you copied this from another project, but I need a reason not to add it now.

Reply:
probably because we don't want to track ssodata? I don't know.

Reply:
@cocojoe after the discussion we had today regarding tracking, is this something you want to track?